### PR TITLE
Change D2k shroud revealing ranges to match original

### PR DESCRIPTION
**Pull request by reaperrr**

### CODE REVIEW

**Comment by penev92** [comment]
You might want to buff up the ConYard sight range a bit - see the start of [this video](https://www.youtube.com/watch?v=0q63U4dFpjQ). I think one additional cell should do it.

And you should definitely buff the vision ranges on both turrets to at least (!) 4. Maybe even 5 or 6.
Also I'm really not convinced about the harvester sight range.

Other than that :+1:

**Comment by reaperrr** [comment]
Updated.

- increased CY sight range by 1 cell.
- increased harvester sight range by 1 cell.
- increased gun turret sight range to 4.75.
- increased missile turret sight range to 5.75.

File: mods/d2k/rules/infantry.yaml [hunk 1]
```diff
@@ -119,8 +125,7 @@ fremen:
 	Health:
 		HP: 700
 	RevealsShroud:
-		Range: 7c0
-	Passenger:
```
Comment (Mailaender) [inline mods/d2k/rules/infantry.yaml]: This looks like a rebase error.

File: mods/d2k/rules/vehicles.yaml [hunk 2]
```diff
@@ -457,9 +457,8 @@ combat_tank_h:
 	Armament:
 		Weapon: 80mm_H
 	Mobile:
-		Speed: 64
-	RevealsShroud:
-		Range: 7c0
+		Speed: 53
+		ROT: 4
```
Comment (Mailaender) [inline mods/d2k/rules/vehicles.yaml]: This changes speed as well. Is this intended or an accident. You didn't mention it in the commit description.

Comment (reaperrr) [inline mods/d2k/rules/vehicles.yaml]: That's a rebase error as well, will fix it.

File: mods/d2k/rules/infantry.yaml (hunk 1 above)
Comment (reaperrr) [inline mods/d2k/rules/infantry.yaml]: I took a look, and it's actually not. Passenger is inherited from `^Infantry`, I just forgot to make this a separate commit or mention it.

**Comment by reaperrr** [comment]
Fixed harkonnen tank speed.

**Comment by reaperrr** [comment]
Updated.

**Comment by obrakmann** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-%28bleed%29/_compare/9beb6aac79b90c38ed03c9a1764c0413bedf3297)
